erpnext/accounts/report/accounts_receivable/accounts_receivable.html

```diff
@@ -185,7 +185,7 @@ <h5 class="text-center">
 
 						{% if(!filters.show_future_payments) { %}
 						<td>
-							{% if(!(filters.party)) { %}
+							{% if(!(filters.party.length)) { %}
```

Comment:
```suggestion
							{% if(!filters.party?.length) { %}
```

The braces are redundant. The `?` ensures that the code still works when `filters` does not have a `party` property.

Reply:
@barredterra Thanks for the suggestion.